tests/test_perturbations.py

```diff
@@ -37,4 +45,23 @@ def test_paraphrase(self):
             model = 'gpt-3.5-turbo',
             temperature = 0.0,
         )
-        print(similar_sentences)
\ No newline at end of file
+        print(similar_sentences)
+
+class TestParaphrase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.num_perturbations = 4
+        self.perturber = Paraphrase(
+            num_perturbations=self.num_perturbations,
+            temperature=0.1,
+        )
+        return
+    
+    def test_paraphrase(self):
+        for prompt in TRUTHFUL_DATASET:
+            sim_prompt = self.perturber.perturb(prompt)
```

Comment:
under the hood the test will hit openai APIs, no ? should we mock out the openai call and return a pre-canned sample response ? i.e. avoid any side effects of calling external APIs in a test